realsense2_camera/src/base_realsense_node.cpp

```diff
@@ -5,6 +5,8 @@
 #include <geometry_msgs/msg/vector3_stamped.hpp>
 #include <rclcpp/clock.hpp>
 #include <fstream>
+#include <inttypes.h>
```

Comment:
What is inttypes? We need the defines in there for something?

Reply:
Leftovers, removed